src/main/resources/edu/cornell/kfs/module/ar/cu-spring-ar.xml

```diff
@@ -77,4 +79,20 @@
     <bean id="arEmailService" parent="arEmailService-parentBean"
         class="edu.cornell.kfs.module.ar.service.impl.CuAREmailServiceImpl" />
 
+    <bean id="cuVerifyBillingFrequencyService" parent="verifyBillingFrequencyService"/>
+    <bean id="verifyBillingFrequencyService-parentBean"
```

Comment:
Shouldn't this override the bean "verifyBillingFrequencyService" instead of "verifyBillingFrequencyService-parentBean"?